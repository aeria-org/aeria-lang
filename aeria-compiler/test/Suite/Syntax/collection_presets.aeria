collection CollectionPresetsTest {
  presets {
    crud
    view
  }
}
