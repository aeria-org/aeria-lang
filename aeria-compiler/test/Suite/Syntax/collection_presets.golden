{
  "kind": "Program",
  "collections": [
    {
      "kind": "Collection",
      "name": {
        "value": "CollectionPresetsTest",
        "kind": "CollectionName"
      },
      "presets": [
        {
          "propertyName": {
            "value": "crud",
            "kind": "PropertyName"
          },
          "kind": "PresetItem"
        },
        {
          "propertyName": {
            "value": "view",
            "kind": "PropertyName"
          },
          "kind": "PresetItem"
        }
      ]
    }
  ]
}
